src/core/uri.js

```diff
@@ -154,21 +128,19 @@ function URI(uri) {
      * console.log(q.b); // logs "2"
      * console.log(q.c); // logs "3"
      */
-    this.getQuery = function () {
-        var vars;
-        var pair;
-        var result = {};
+    getQuery() {
+        const result = {};
 
         if (this.query) {
-            vars = decodeURIComponent(this.query).split("&");
-            vars.forEach(function (item, index, arr) {
-                pair = item.split("=");
+            const queryParams = decodeURIComponent(this.query).split("&");
+            for (const queryParam of queryParams) {
```

Comment:
Cool I've been using `forEach`, but seems like `for of` is better (in most cases) as it doesn't skip empty elements.